CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ Changes
 
 **New features**
 
+* [#1637] Added 2FA which can be disabled by the environment variable``DISABLE_2FA``.
```

Comment:
This feature would be added to the next release (I think 1.7.0), not the existing one